packages/cloud-cognitive/src/components/EditTearsheet/EditTearsheet.mdx

```diff
@@ -0,0 +1,190 @@
+import { Story, ArgsTable, Canvas } from '@storybook/addon-docs';
+import {
+  getStoryId,
+  CodesandboxLink,
+} from '../../global/js/utils/story-helper';
+import { CreateTearsheet } from '../CreateTearsheet';
+import { EditTearsheet } from '.';
+import createTearsheetGif from '../CreateTearsheet/storybook_assets/create_tearsheet.gif';
+import createTearsheetAnatomy from '../CreateTearsheet/storybook_assets/create_tearsheet_anatomy.jpg';
+
+# EditTearsheet
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+- [Component API](#component-api)
+
+## Overview
+
+When to use
+
+- Use a wide tearsheet with creations that have two or more distinct steps.
+- Omit the progress indicator to use with single step creations that require
+  more space, such as those with interactive selections, diagrams, or tables.
+
+When not to use
+
+- Don’t use the wide tearsheet for low complexity create flows with only one
+  step.
```

Comment:
Should also update this with the [EditTearsheet version](https://pages.github.ibm.com/cdai-design/pal/patterns/edit/usage#wide-tearsheet-edit) of this.